lib/generator/template_values.rb

```diff
@@ -1,24 +1,30 @@
 module Generator
   # Contains methods accessible to the ERB template
   class TemplateValues
-    attr_reader :abbreviated_commit_hash, :version, :test_cases
+    attr_reader :abbreviated_commit_hash, :version, :exercise_name, :test_cases
```

Comment:
I had intended to isolate adding `exercise_name` into it's own commit, but was trying to get this pushed in time for you to review :)